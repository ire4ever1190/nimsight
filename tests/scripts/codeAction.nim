#> :w
#> :wait textDocument/publishDiagnostics

proc bamboo() = discard

bambo() #[
  ^ :CodeAction ]#


# Imports
import strutils #[
        ^ :CodeAction ]#
import std/[parseopt, strformat, macrocache] #[
             ^ :CodeAction ]#
import std/[parseopt, strformat, macrocache] #[
                        ^ :CodeAction ]#
import std/[parseopt, strformat, macrocache] #[
                                    ^ :CodeAction ]#
import strutils, macros #[
          ^ :CodeAction ]#


#> :wait TextChanged
