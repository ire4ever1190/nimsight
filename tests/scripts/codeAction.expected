#> :w
#> :wait textDocument/publishDiagnostics

proc bamboo() = discard

bamboo() #[
  ^ :CodeAction ]#

#> :wait TextChanged
